src/sagemaker/estimator.py

```diff
@@ -357,7 +357,7 @@ def __init__(
                 monitoring and profiling, set the
                 ``disable_profiler`` parameter to ``True``.
             disable_profiler (bool): Specifies whether Debugger monitoring and profiling
-                will be disabled (default: ``False``).
+                will be disabled (default: ``None``).
```

Comment:
Ah so this we want to keep as False :D 
Because if the user passes in nothing, and nothing is set in the config, the constructor will still set it to False, (but only after checking the config, whereas before it was done thru the constructor default value)

Reply:
Ahhhh Makes sense - Sorry misunderstood your previous comment - Thanks for the catch :)